recipes/pytorch-tabnet/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pytorch-tabnet" %}
+{% set version = "3.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch_tabnet-{{ version }}.tar.gz
+  sha256: 69f5fb07dc411abc3f341ad41ba7053e98ac028950badd92c6838939b1cadcbf
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry >=0.12
+  run:
+    - python >=3.6,<4.0
```

Comment:
let's remove the python upper bound. We may need this eventually, but this will need to be done globally.

Reply:
I included that as it was mentioned in the **PKG-INFO** file.

![image](https://user-images.githubusercontent.com/10201242/146639844-634c7043-9b2a-498c-9442-1685fab42410.png)

Reply:
yes people do that but we have to manage things globally in conda-forge.

Reply:
Done.

Reply:
> We may need this eventually, but this will need to be done globally.

@beckermr What I am curious about is, how/why does this upper bound impact the current setup? What could potentially break if this upper-bound is left as is?

Reply:
It is not that things would break, but that we're going to have to add that bound globally. We don't know if python 4, whenever it exists, will be incompatible with python 3 or not. If it is not, then we don't need it. If it is, then we'll have to add it. Having to go back and remove the bound is also a pain. So once we know, we'll patch the whole ecosystem at once.